<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="User-PC" name="AttemptAll1" tests="1" failures="1" timestamp="11 May 2017 02:47:42 GMT" time="29.387" errors="0">
  <testcase name="SurveyData1" time="29.384" classname="Student.AttemptSurvey">
    <failure type="java.lang.AssertionError" message="Login Failed for:mm.student1@inpods.com expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login Failed for:mm.student1@inpods.com expected [true] but found [false]
	at Common.Login.LoginD(Login.java:35)
	at Common.Login.main(Login.java:89)
	at Student.AttemptSurvey.AttemptSurvey(AttemptSurvey.java:65)
	at Student.AttemptSurvey.SurveyData1(AttemptSurvey.java:38)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- SurveyData1 -->
</testsuite> <!-- AttemptAll1 -->
